cmd/checkmarxExecuteScan_generated.go

```diff
@@ -398,14 +399,6 @@ func checkmarxExecuteScanMetadata() config.StepData {
 						Mandatory:   false,
 						Aliases:     []config.Alias{},
 					},
-					{
```

Comment:
Why does this verbose flag here disappear with a pull request having its focus on telemetry?

Reply:
Looks like the previous PR didn't update it's generated sources based on the lasted yaml file.

Reply:
Looks like the previous PR didn't update it's generated sources based on the lasted yaml file.